### ISSUE TITLE
sc-alert class light wrong color

### ISSUE THREAD

**Issue by fugo** [pmoreno-rodriguez/grav-theme-mundana#15]
By using in ``[sc-alert]`` the class ``light`` the text color is **white** and then invisible.  
I found in theme.css and theme.min.css the wrong declarion and changed it to ``color: #818182;``.

```css
.alert-light {
	color:#fff;
	background-color:#f8f9fa;
	border-color:#f8f9fa;
}
```

**Comment by pmoreno-rodriguez** [comment]
Mundana original theme seems has modified the bootstrap CSS for some alerts. In this case the CSS classes for alert light are: 
```
.alert-light {
  color: #818182;
  background-color: #fefefe;
  border-color: #fdfdfe;
}
.alert-light hr {
  border-top-color: #ececf6;
}
.alert-light .alert-link {
  color: #686868;
}
```
Well, I'll try to modify this for the next version of theme. 

Thanks.

**Comment by pmoreno-rodriguez** [comment]
Fixed on release 1.3.1
